Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -244,6 +248,51 @@ private void ApplyConfigurations()
             MixedRealityProjectConfigurator.ConfigureProject(configurationFilter);
         }
 
+        /// <summary>
+        /// Provide the user with the list of spatializers that can be selected.
+        /// </summary>
+        private void PromptForAudioSpatializer()
+        {
+            string selectedSpatializer = MixedRealityProjectConfigurator.SelectedSpatializer;
+            List<string> spatializers = new List<string>();
+            spatializers.Add("None");
```

Comment:
Avoid using hardcoded strings in multiple locations - if we're going to be using "None" as a special value, have this be a const earlier in the class (because on line 286 we compare against "None" directly).

Reply:
Refactored out the const